CharlieBackend.AdminPanel/Models/Calendar/CalendarEventOccurrenceViewModel.cs

```diff
@@ -14,5 +14,7 @@ public class CalendarEventOccurrenceViewModel
         public DateTime EventFinish { get; set; }
 
         public PatternType Pattern { get; set; }
+
+        public long Storage { get; set; }
```

Comment:
Where are you using this property?

Reply:
Removed. 9addd8f7c3c858f432346be32c39219df9043ca0